app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,10 @@
             android:theme="@style/AppTheme"
             android:usesCleartextTraffic="true">
         <activity android:name="com.yalta.activities.DriverMainActivity" android:screenOrientation="portrait" />
-        <activity android:name="com.yalta.activities.AdminMainActivity" android:screenOrientation="portrait" />
+        <activity
+            android:name="com.yalta.activities.AdminMainActivity"
+            android:screenOrientation="portrait"
+            android:windowSoftInputMode="adjustPan" />
```

Comment:
Maybe move this back to activity definition? it's weird that it's the only activity here that has it

Reply:
But this parameter can only be set in Manifest...